developers/wcs/tools/collections-tool.mdx

```diff
@@ -1,77 +1,116 @@
+#### Select multi-tenancy
 
-### Pass authentication details
+[Multi-tenant collections](/developers/weaviate/manage-data/multi-tenancy) are useful if you have multiple users or need data isolation. By default, collections are single tenant. Set the `Multi-tenancy` toggle to enable multi-tenancy.
 
-import ExternalAuth from '/_includes/wcs/query-auth-details.mdx';
+#### Set a default vectorizer
 
-<ExternalAuth/>
+Set a [vectorizer](/developers/weaviate/config-refs/schema#vectorizer) for the collection. Weaviate uses the default vectorizer when you import unvectorized data and when Weaviate queries external APIs that require vectorized inputs.
 
-### Pass inference keys
+The `Vectorizer` setting is a drop-down list. After you select a vectorizer, the editor expands to let you configure the model.
 
-To pass API keys for inference modules, use request headers. The `Header` tab is at bottom of the query screen. Add the service inference key to the query headers.
+Switch the `Vectorize class name` toggle if you don't want to create a vector for the class name.
 
-In this example, the `X-OpenAI-Api-Key` header sets the OpenAI API key.
+### Edit the Properties section
 
-<center><img src={WCDInferenceApiKey} width="100%" alt="How to add an API inference key to a WCD console query"/></center>
+Use the `Properties` section to fine-tune individual properties in your collection.
 
-The header can pass multiple keys.
+#### Add properties to the collection
 
-<center><img src={WCDManyInferenceApiKeys} width="80%" alt="How to add multiple API inference keys to a WCD console query"/></center>
+Enter property names into the properties editor.
+
+The property names must begin with a lowercase letter.
+
+The names can only contain letters, numbers, and the under-score (_) character. Spaces are not allowed.
+
+<center><img src={WCDEdProp} width="75%" alt="Properties edit section"/></center>
+
+#### Configure additional details
+
+Some property types can be fine tuned. When additional configuration is possible, the editor shows a gear wheel next to the `Filterable` toggle.
+
+To update these additional settings, click on the gear wheel then select the configuration options in the pop-up window.
+
+<center><img src={WCDEdExProp} width="50%" alt="Properties additional details edit section"/></center>
+
+## Modify a collection
+
+To modify a collection, click the `Collection` option, then select the cluster that hosts the collection that you want to modify. Click the pencil icon that is located near the collection name to open the collection for editing.
+
+<center><img src={WCDEdSelect} width="100%" alt="Select to edit or delete"/></center>
+
+To add a new property to the collection, enter the name and type in the editor. Update additional configuration details if the property type supports them.
```

Comment:
Only top-level properties can be added. We do not support adding new nested properties.